OpenTrons/opentrons-api
# feat(api): Implement `define_liquid` and `load_liquid` in PAPI

# Overview

closes https://opentrons.atlassian.net/browse/RLIQ-7.
Add a `define_liquid` and `load_liquid` in PAPI.

# Test Plan

- Test against an OT-2\OT-3 with core ff on.
- Upload a python protocol to test against PAPI. 

```
from opentrons import protocol_api, types

metadata = {
    "apiLevel": "2.13",
}

def run(ctx: protocol_api.ProtocolContext) -> None:
    labware = ctx.load_labware("nest_96_wellplate_100ul_pcr_full_skirt", 5)
    liquid = ctx.define_liquid(name="water", description="", display_color="#fff")
    well = labware.wells()[0]
    well.load_liquid(liquid=liquid, volume=20.0)
```
- Make sure the robot analysis passes. 
- Test with ff off. Make sure you are getting an unsupported exception. 

# Changelog
- added `define_liquid` to `protocol_context` and to protocol core's implementation.
- added `load_liquid` to `well_context` and to well core's implementation. 
- added `load_liquid` to `sync_client`.
- added `add_liquid` to `sync_client`.

# Review requests

1. did I miss anything?
2. are test sufficient? 

# Risk assessment

Low. added a new end points in the PAPI to `define_liquid ` and `load_liquid`.
